iree/turbine/kernel/wave/utils/run_utils.py

```diff
@@ -242,15 +242,16 @@ def invoke_with_wave_runtime(
     Invokes the kernel with the wave runtime.
     """
     import wave_runtime
-    from ..cache import WAVE_RUNTIME_DIR, CACHE_BASE_DIR
+    from ..cache import get_wave_runtime_dir, get_cache_base_dir
 
     # Get the path to the binary.
     if options.kernel_hash:
         binary = (
-            str(CACHE_BASE_DIR / options.kernel_hash / options.kernel_hash) + ".hsaco"
+            str(get_cache_base_dir() / options.kernel_hash / options.kernel_hash)
+            + ".hsaco"
         )
```

Comment:
drive-by: Does Wave _only_ support hsaco? IREE (and iree-turbine) support multiple targets and kernel formats. Could use a generic file extension or a container format like IREE's VMFB (or other flatbuffers: https://github.com/iree-org/iree/tree/main/runtime/src/iree/schemas).

This could also use [`PurePath.with_suffix`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.with_suffix) instead of appending to a string.

Reply:
Standalone wave runtime only supports hip/hsaco currently. IREE-based runtime doesn't need this code, but it has perf problems with dlpack